contrib/systemd/matrix-synapse.service

```diff
@@ -23,7 +23,7 @@ User=synapse
 Group=nogroup
 
 WorkingDirectory=/opt/synapse
-ExecStart=/opt/synapse/env/bin/python -m synapse.app.homeserver --config-path=/opt/synapse/homeserver.yaml
+ExecStart=/opt/synapse/env/bin/python3 -m synapse.app.homeserver --config-path=/opt/synapse/homeserver.yaml
```

Comment:
Alright, if a distro only had a `python3` binary in the environment, then it's probably fine to update it to that instead of `python`.

It would be nice to add a comment above to state that the path to the python binary is likely to need replacing.

Reply:
Hrm, perhaps I didn't properly validate the contents of the env. I do have a python link to points to the environment python3. I wonder if I had a typo initially? Eitherway, my shame is real.

Here are the `python*` listings of my `synapse/env/bin`:

```[synapse@XXX bin]$ pwd
/home/synapse/synapse/env/bin
[synapse@XXX bin]$ ll python*
lrwxrwxrwx. 1 synapse synapse     7 Oct 24 16:59 python -> python3
-rwxrwxr-x. 1 synapse synapse 11408 Oct 24 16:59 python3
lrwxrwxrwx. 1 synapse synapse     7 Oct 24 16:59 python3.6 -> python3
-rwxrwxr-x. 1 synapse synapse  2343 Oct 24 16:56 python-config
```
Not sure why that didn't work when I previously updated my path with the unit file. This basically nullifies my change as it should have worked, but I can chalk this up to fat fingers I suppose. 

I'll submit a separate PR for the comments concerning paths according to the installation instructions and close this one. Sorry for burning review time :disappointed:

Reply:
Huh ok. No worries, mistakes happen. All part of the fun :)